internals/overlord/servstate/manager_test.go

```diff
@@ -525,24 +395,28 @@ func (s *S) TestCurrentUserGroup(c *C) {
 services:
     usrtest:
         override: replace
-        command: /bin/sh -c "id -n -u >%s; sleep %g"
+        command: /bin/sh -c "id -n -u >%s; sync; touch %s; sleep %g"
         user: %s
         group: %s
-`, outputPath, shortOkayDelay.Seconds()+0.01, current.Username, group.Name))
-	err = manager.AppendLayer(layer)
+`, outputPath, s.donePath, shortOkayDelay.Seconds()+0.01, current.Username, group.Name))
+	err = s.manager.AppendLayer(layer)
 	c.Assert(err, IsNil)
 
 	chg := s.startServices(c, []string{"usrtest"}, 1)
 	s.st.Lock()
 	c.Assert(chg.Err(), IsNil)
 	s.st.Unlock()
 
+	// Let's wait until we know the output file is available
+	s.waitForFileToAppear(c)
```

Comment:
What about calling this `waitForDoneFile`, so its not ambiguous what file it's waiting for? And we probably don't need this comment every time the function is called -- a clear name should be enough.

Reply:
Like your proposal - applying everywhere.